packages/search-ui-app-search-connector/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "Apache-2.0",
   "main": "lib",
   "module": "es",
+  "types": "search-ui-app-search-connector.d.ts",
```

Comment:
small remark: I haven't seen this before when adding type declarations - usually it's just an `index.d.ts` file in the root directory